MdeModulePkg/Library/CustomizedDisplayLib/CustomizedDisplayLib.c

```diff
@@ -155,8 +157,9 @@ RefreshKeyHelp (
     return;
   }
 
-  SecCol            = gScreenDimensions.LeftColumn + (gScreenDimensions.RightColumn - gScreenDimensions.LeftColumn) / 3;
-  ThdCol            = gScreenDimensions.LeftColumn + (gScreenDimensions.RightColumn - gScreenDimensions.LeftColumn) / 3 * 2;
+  SecCol            = gScreenDimensions.LeftColumn + (gScreenDimensions.RightColumn - gScreenDimensions.LeftColumn) / 4;
+  ThdCol            = gScreenDimensions.LeftColumn + (gScreenDimensions.RightColumn - gScreenDimensions.LeftColumn) / 4 * 2;
+  FthCol            = gScreenDimensions.LeftColumn + (gScreenDimensions.RightColumn - gScreenDimensions.LeftColumn) / 4 * 3;

```

Comment:
With 4th column it might be pretty tight with 80x25 console resolution. You would have to check it on some protectli platform for example.